### ISSUE TITLE
Vendoring latlong with godep

### ISSUE THREAD

**Issue by pshevtsov** [tools/godep#228]
I'm trying to vendor [latlong] dependencies with Godeps. But when I run

```bash
$ godep save
```

I get the following error message:

```
godep: Godeps/_workspace/src/github.com/bradfitz/latlong/z_gen_tables.go:23:18: expected '}', found 'EOF'
```

[latlong]: https://github.com/bradfitz/latlong

**Comment by freeformz** [comment]
Godep is truncating data, probably because the line being truncated is longer than 64k. If I'm right I may have a fix soon.

**Comment by freeformz** [comment]
This should now be fixed by #231, so please `go get -u github.com/tools/godep` and re-open if this is still a problem (PS: You may need to cleanup your broken workspace first).
